pkg/controller/capability/capability.go

```diff
@@ -77,11 +73,8 @@ func (in *Capability) SetInitialStatus(msg string) bool {
 }
 
 func (in *Capability) CheckValidity() error {
```

Comment:
What will happen if the capability needed is not there, the status of the Capability will include a comment, will the Component be updated to also include the status's comment ?

Reply:
I haven't tested that use case yet as it was already difficult enough to get it working for the working case… :)

Reply:
If the plugin is not available, the capability will be set to `Failed` and show an error message.